libzim/libzim.pyx

```diff
@@ -1329,19 +1483,57 @@ cdef class Archive:
             return self.c_archive.hasIllustration(size)
         return self.c_archive.hasIllustration()
 
-    def get_illustration_item(self, size: pyint = None) -> Item:
+    def get_illustration_item(self, size: pyint = None, info: IllustrationInfo = None) -> Item:
         """Get the illustration Metadata item of the archive.
 
+        Args:
+            size: Optional size of the illustration (for backward compatibility).
```

Comment:
If optional, the default should be mentioned

Reply:
It is optional in the sense that you provide either directly the size or the full info. It is anyway not consistent with add_illustration, and I prefer when there is one single arg to match libzim, so let's move to this.